pkgs/games/xu4/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, zlib
+, libpng
+, boron
+, allegro5
+, libGL
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xu4";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "xu4-engine";
+    repo = "u4";
+    rev = "v${version}";
+    sha256 = "sha256-9rsBsC325P/oLjTh8nX2Qo7Yxkl+w6d7OYjFYXuiZrs=";
+  };
+
+  # this is not a standard Autotools-like `configure` script
+  dontAddPrefix = true;
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
+
+  postConfigure = ''
+    # need to report this upstream
+    sed --in-place 's/-Wall/-Wno-error=format-security/g' src/Makefile
+  '';
+
+  enableParallelBuilding = true;
+
+  buildFlags = [ "src/xu4" "Ultima-IV.mod" ];
+
+  buildInputs = [
+    zlib
+    libpng
+    boron
+    allegro5
+    libGL
+    libxml2
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # there's no `install` target
+  installPhase =
+  ''
+    install -D -m 755 src/xu4 $out/share/xu4
+    install -D -m 644 Ultima-IV.mod $out/share/Ultima-IV.mod
+    install -D icons/xu4.png $out/share/icons/hicolor/48x48/apps/xu4.png
+    makeWrapper $out/share/xu4 $out/bin/xu4 --chdir $out/share
+  '';
+
+  desktopItems = [(makeDesktopItem {
+    name = "xu4";
+    exec = "xu4";
+    icon = "xu4";
+    comment = "Ultima IV engine";
+    desktopName = "xu4";
+    categories = [ "Game" ];
+  })];
+
+  meta = with lib; {
+    homepage = "http://xu4.sourceforge.net/";
```

Comment:
```suggestion
    homepage = "https://xu4.sourceforge.net/";
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/181866/commits/2b01e149e8d64fae71ad3c7c9b58b39b0b7a5201

Reply:
You reverted this change later on

Reply:
oops, reapplied in https://github.com/NixOS/nixpkgs/pull/181866/commits/467e54a9d3a81db5f064aa834b44eead13124b59